internal/resp_assertions/string_array_assertion.go

```diff
@@ -2,16 +2,18 @@ package resp_assertions
 
 import (
 	"fmt"
+	"sort"
 
 	resp_value "github.com/codecrafters-io/redis-tester/internal/resp/value"
 )
 
 type StringArrayAssertion struct {
-	ExpectedValue []string
+	ExpectedValue              []string
+	shouldSortBeforeComparison bool
```

Comment:
```suggestion
	ShouldSortBeforeComparison bool
```

(If `ExpectedValue` is public, no reason for this to not be)